src/actions/errorAction.test.js

```diff
@@ -1,11 +1,17 @@
-import configureMockStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
-import * as actions from './incidentAction';
-import * as types from './actionTypes';
+// third-party libraries
```

Comment:
What's happening on this line @cosmas28

Reply:
It's a comment, do you mean I should remove it?